dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -150,6 +150,27 @@ const textStandfirst = (format: ArticleFormat): string => {
 	return BLACK;
 };
 
+const textLastUpdatedStandfirst = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.LiveBlog) {
+		switch (format.theme) {
+			case ArticlePillar.News:
+				return news[600];
+			case ArticlePillar.Culture:
+				return culture[600];
+			case ArticlePillar.Lifestyle:
+				return lifestyle[600];
+			case ArticlePillar.Sport:
+				return sport[600];
+			case ArticlePillar.Opinion:
+				return opinion[600];
+			case ArticleSpecial.Labs:
+			case ArticleSpecial.SpecialReport:
+				return news[600];
```

Comment:
Thanks Oliver. I will chat with Ben about this and update it then 👍

Reply:
I had a chat with Ben and suggested that because we don't yet have a proper design for special report liveblogs, until we have such a design, the fallback in composer is news palette. And so, news[600] would be the colour to use for the pulsating dot

Reply:
The problem with this is someone can, and at some point will, launch a special report liveblog and so either DCR supports this theme or we need to rollback support for it

@jamesgorrie In the absence of designs for this theme, should we revert support for DCR and return to having them rendered by Frontend or do we want to move forward on this?

Reply:
Note. I think when we added the default to `news` in other places before that was when DCR was only rendering a small percentage of blogs. But now that we're at 100% I'm not sure this is a good approach.

Reply:
We discussed this internally and currently, these types of live blogs are [**not supported by DCR**](https://github.com/guardian/frontend/blob/120da6027da932b3f7036091fc61b5971af6942e/article/app/controllers/LiveBlogController.scala#L348-L355), so it’s not an issue yet.